src/st_stuff.c

```diff
@@ -441,24 +441,7 @@ void ST_refreshBackground(boolean force)
 
     // [crispy] copy entire video.unscaledw, to preserve the pattern to the left
     // and right of the status bar in widescren mode
-    if (!force)
```

Comment:
I don't even remember why this branch was here?

Reply:
There is another `V_CopyRect` in `ST_drawWidgets`, so it's not necessary to copy the ST_BAR itself in case of `R_ExecuteSetViewSize`. I don't think we need this complication.